docs/CC2023/README.md

```diff
@@ -0,0 +1,74 @@
+
+# MOD2IR: High-Performance Code Generation for a Biophysically Detailed Neuronal Simulation DSL
```

Comment:
I think that the attachment of the artifact can be only this file? Or should we provide the docker image as well?

@georgemitenkov @ohm314 @pramodk